.github/workflows/build-pr-preview.yml

```diff
@@ -269,12 +279,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Checking links in files
+        run: |
+          echo "Checking links in files:"
+          echo "${{ needs.get-changed-files.outputs.changed_markdown_content_files }}" | tr ' ' '\n'
```

Comment:
Even though we can see the files checked in "Run lychee link checker", they are in the args so a bit hard to read. This fixes that.